source/simulator/assemblers/stokes.cc

```diff
@@ -229,11 +229,15 @@ namespace aspect
               data.local_rhs(i) += (density * gravity * scratch.phi_u[i])
                                    * JxW;
 
-              if (force != NULL)
+              if (force != NULL && this->get_parameters().enable_additional_stokes_rhs)
```

Comment:
If you do what @gassmoeller describes below, you don't need this change anymore.